docusaurus.config.js

```diff
@@ -49,204 +49,156 @@ module.exports = {
         {
           to: "Overview",
           label: "Overview",
-          position: "left",
+          
         },
         {
+          to: "Overview",
           label: "Modules",
-          position: "left",
+          
           items: [
             {
               href: "https://introtocoding.codeyourfuture.io",
               label: "Intro To Coding",
-              position: "left",
-              className: "outside-link",
```

Comment:
![image](https://user-images.githubusercontent.com/5181870/145731308-8c7e9f57-4a92-412c-a842-8795db48a4b5.png)

These classNames add the nice little "this link opens another website" styling to the buttons. It's nice to have but if ti means the config won't compile then I guess it'll have to go

Reply:
It does it anyway now, for some reason.